test/unit/cmds/login.test.js

```diff
@@ -6,74 +6,68 @@ import {
   handler as loginHandler,
```

Comment:
these tests were failing (but only on TravisCI and not locally 🙈) because of some changes in `login.js` related to `process.platform` etc. I did some pretty major refactoring here to stub more things directly in login.js (and not context.js)